org.fsnebula.Knossos.yml

```diff
@@ -0,0 +1,68 @@
+app-id: org.fsnebula.Knossos
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: knossos
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+  - --share=network
+  - --filesystem=host
+build-options:
+    append-path: /usr/lib/sdk/node18/bin:/usr/lib/sdk/rust-stable/bin
+    env:
+        XDG_CACHE_HOME: /run/build/knossos/flatpak-node/cache
+        npm_config_nodedir: /usr/lib/sdk/node18
+modules:
+    - python3-requirements.json
+    - p7zip.yml
+    - name: fuseless
+      buildsystem: simple
+      build-options:
+        env:
+          CARGO_HOME: /run/build/fuseless/cargo
+      build-commands:
+        - cargo --offline fetch --manifest-path Cargo.toml --verbose
+        - cargo --offline build --release --verbose
+        - install -Dm755 target/release/libfuseless.so /app/lib/
+        - ln -s /app/lib/libfuseless.so /app/lib/libfuse.so.2
+      sources:
+        - type: git
+          path: /home/phantomhoover/code/fuseless
```

Comment:
definitely not.

Reply:
This line wasn't changed. Why this conversation was marked as resolved when it's clearly not?